### PR TITLE
[ISSUE-690][DOC] Storage resource quota doc

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### Related issues.
https://github.com/alibaba/celeborn/issues/690

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: docs/storage-usage-quota.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+---
+
+Storage Usage Quota Guide
+===
+This documentation contains how RSS limit user's storage resource usage by setting quota.
```
Comment (waitinfuture) [inline docs/storage-usage-quota.md:22]: contains -> describes
limit -> limits

File: docs/storage-usage-quota.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Quota limitation
+
+RSS has a configurable user storage quota system. This allows RSS admin to manage each user's
+max resource usage to keep RSS cluster more stable. This feature can avoid RSS cluster resources
+been occupied by several users with huge application.
```
Comment (waitinfuture) [inline docs/storage-usage-quota.md:28]: been -> being
several users -> the minority
application -> applications

File: docs/storage-usage-quota.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+RSS cluster's worker will collect each user's resource consumption and report these information
```
Comment (waitinfuture) [inline docs/storage-usage-quota.md:30]: these -> the

File: docs/storage-usage-quota.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+will check current user's resource usage, if used resource exceed the quota setting,
+shuffle will be fallback to ESS.
+
+## Storage resource
+
+Currently, RSS support two storage level:
+  1. Local disk.
+  2. HDFS.
+
+And there are two level of resources:
```
Comment (waitinfuture) [inline docs/storage-usage-quota.md:41]: level -> levels

File: docs/storage-usage-quota.md (hunk 1 above)
Comment (AngersZhuuuu) [reply docs/storage-usage-quota.md:22]: Done

File: docs/storage-usage-quota.md (hunk 2 above)
Comment (AngersZhuuuu) [reply docs/storage-usage-quota.md:28]: Done

File: docs/storage-usage-quota.md (hunk 4 above)
Comment (AngersZhuuuu) [reply docs/storage-usage-quota.md:41]: Done

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**